backend/lucene/src/main/java/org/hibernate/search/backend/lucene/index/impl/ShardHolder.java

```diff
@@ -50,7 +50,7 @@ public String toString() {
 		return getClass().getSimpleName() + "[indexName=" + model.getIndexName() + "]";
 	}
 
-	CompletableFuture<?> start(IndexManagerStartContext startContext) {
+	void start(IndexManagerStartContext startContext) {
```

Comment:
That's nice. I would like to do the same whenever it would be possible.

Reply:
For sure. I don't use reactive programming for my own entertainment ;D